etc/conda-activate.sh

```diff
@@ -2,9 +2,10 @@
 
 set -Ee
 
-rm -rf "$CONDA_PREFIX"/include/{rmm,cudf,libcudf,cuml,cugraph,cuspatial}
+rm -rf "$CONDA_PREFIX"/include/{rmm,kvikio,cudf,libcudf,cuml,cugraph,cuspatial}
```

Comment:
Missing raft here? Also unclear why libcudf is listed but no other "lib" names.

Reply:
I think libcudf is an error and you're right, raft should be included.